### PR TITLE
Patch Batch Autogen specification

### PR DESCRIPTION
**Pull request by bgklein**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/batch/data-plane/readme.md [hunk 1]
```diff
@@ -305,7 +305,7 @@ python:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-batch/azure/batch
+  output-folder: $(python-sdks-folder)/sdk/azure-batch/azure/batch
```
Comment (lmazuel) [inline specification/batch/data-plane/readme.md]: Should be
```
output-folder: $(python-sdks-folder)/batch/azure-batch/azure/batch
```
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/batch

**Comment by matthchr** [comment]
Should we delete this?
https://github.com/Azure/azure-rest-api-specs/blob/7dda925fbc16796fd9938cdcb1d54ec6fef47c3a/specification/batch/data-plane/readme.nodejs.md

**Comment by bgklein** [comment]
Yea Matt I'll try to revisit this by EOW

**Comment by lmazuel** [comment]
Closing, since I believe we should start over (and we have a meeting next Tuesday with Dave and team)

**Comment by JackTn** [comment]
Hi, @bgklein. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Comment by JackTn** [comment]
Hi, @bgklein. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @bgklein. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @bgklein. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**